Core/MIPS/ARM/ArmAsm.cpp

```diff
@@ -70,9 +71,84 @@ namespace MIPSComp {
 
 using namespace ArmJitConstants;
 
-void ArmJit::GenerateFixedCode()
-{
-	enterCode = AlignCode16();
+void ArmJit::GenerateFixedCode() {
+	const u8 *start = GetCodePtr();
+
+	// LR == SCRATCHREG2....  Maybe we could find some other register to use to avoid having to
+	// push LR?
```

Comment:
Then again, isn't POP R_PC the preferred way to return?  Maybe it's good, and the PUSH of SCRATCHREG1 should combine, right?

-[Unknown]